src/runtime/google-analytics-event-listener.js

```diff
@@ -73,13 +73,13 @@ export class GoogleAnalyticsEventListener {
       eventLabel: analyticsParams.event_label || gaEvent.eventLabel,
     };
     // TODO(b/234825847): Remove it once universal analytics is deprecated in 2023.
-    const ga = this.win_.ga || null;
-    if (isFunction(ga)) {
+    if (this.constructor.isGaEligible(this.deps_)) {
+      const ga = this.deps_.win().ga || null;
       ga('send', 'event', gaEvent);
```

Comment:
If `ga` is null, this would throw an error, right?

Reply:
It shouldn't actually ever be null due to the `isGaEligible` check; is there a way to satisfy the type check? Since `ga`/`gtag` are not defined on the `Window` type, without the `null` fallback it complains:
```
ERROR - [JSC_INEXISTENT_PROPERTY] Property gtag never defined on Window

Reply:
You could try casting the type with inline JSDoc comments?

Reply:
Neat, how is this?

Reply:
I was thinking something more like this, when you declare the `ga` variable

```
const ga = /** @type {!function(string, string, Object)} */ (this.deps_.win().ga);
```

Could that work?

Reply:
I believe it's the `Window` type that is the problem: `win().ga`. This suggested change still throws the same error `Property ga never defined on Window`

Reply:
Good call! I just made some suggestions here via a PR https://github.com/justinchou-google/swg-js/pull/1/

Reply:
Thanks, LGTM